src/sentry/utils/http.py

```diff
@@ -156,3 +158,26 @@ def is_valid_origin(origin, project=None, allowed=None):
         if parsed.path.startswith(path):
             return True
     return False
+
+
+def is_valid_ip(ip_address, project):
+    """
+    Verify that an IP address is not being blacklisted
+    for the given project.
+    """
+    blacklist = project.get_option('sentry:blacklisted_ips')
```

Comment:
Nope seems fine. That said I have been aiming to use less underscores and more hyphens lately as they are less awful.